TLSharp.Tests.VS/TLSharpTestsVs.cs

```diff
@@ -0,0 +1,71 @@
+﻿
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TLSharp.Tests
+{
+    [TestClass]
+    public class TLSharpTestsVS : TLSharpTests
```

Comment:
I also thought about doing it this way (inversely) but this is not possible because MSTest-related assemblies are not loaded at all in IDEs which are not Visual Studio. Therefore, you wouldn't be able to compile TLSharpTestsNUnit because it wouldn't compile the referenced project.